lib/db/schema/patch-085-086.sql

```diff
@@ -0,0 +1,56 @@
+-- This migration updates recoveryKey table to store a hashed value of
+-- the `recoveryKeyId` instead of the raw value. As part of this migration,
+-- we will remove all existing recoveryKeys. This will help to ensure a consistent
+-- migration.
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('85');
+
+DELETE FROM recoveryKeys;
+
+-- We can drop this since we will be using the `recoveryKeyIdHash`
+-- column instead.
+ALTER TABLE recoveryKeys DROP COLUMN recoveryKeyId,
+ALGORITHM = INPLACE, LOCK = NONE;
```

Comment:
If someone tries to add a recovery key while we're in the process of rolling this out, they might end up calling the old `createRecoveryKey_2` procedure, which will error out trying to access the now-removed `recoveryKeyId` column.

That's probably for the best (and is better than letting them insert a soon-to-not-be-working-correctly key`) but we should document it here explicitly to re-assure our friendly Ops folks that it's safe to drop this column :-)